lldb/include/lldb/Host/XML.h

```diff
@@ -11,6 +11,11 @@
 
 #include "lldb/Host/Config.h"
 
+#if defined(__AIX__)
+//FIXME for AIX
+#undef LLDB_ENABLE_LIBXML2
```

Comment:
In theory if you configure with `-DLLDB_ENABLE_LIBXML2=OFF`, the libxml2 parts will be removed. Was something else being included despite that?